src/frontends/onnx/docs/how_to_add_op.md

```diff
@@ -95,7 +85,7 @@ If an OpenVINO Core operation provides exactly what you need (without decomposit
 ```cpp
 core.add_extension(ov::frontend::onnx::OpExtension<ov::opset9::Add>("org.openvinotoolkit", "CustomAdd"));
 ```
-If you need to register an custom operation for [OpenVINO Model Converter](../../../../tools/ovc) scenario, you should consider `SOExtension`. More details about it can be found in [Library with Extensions](../../../../docs/Extensibility_UG/Intro.md#create-a-library-with-extensions).
+If you need to register an custom operation for a [Model Optimizer](../../../../tools/mo) scenario, you should consider `SOExtension`. More details about it can be found in [Library with Extensions](../../../../docs/Extensibility_UG/Intro.md#create-a-library-with-extensions).
```

Comment:
```suggestion
If you need to register a custom operation for a [Model Optimizer](../../../../tools/mo) scenario, you should consider `SOExtension`. More details about it can be found in [Library with Extensions](../../../../docs/Extensibility_UG/Intro.md#create-a-library-with-extensions).
```
Since Model Optimizer has been deprecated with 2025.0, is this part correct?

Reply:
Good question, I'll check. I have an expectation it should be aligned with ovc...

Reply:
Removed this part